tests/nested/manual/core20-boot-config-update/task.yaml

```diff
@@ -0,0 +1,155 @@
+summary: Check that the boot config is correctly updated when snapd is refreshed
```

Comment:
why is it moved to manual?

Reply:
The image is built with customized gadget, whereas the nested core suite tests execute with a standard pc snap that that only got the shim signed by the test setup.